src/modules/claim/claim.controller.ts

```diff
@@ -83,14 +78,11 @@ export class ClaimController {
 
     await validateOrReject(claimDTO);
 
-    const { sub } = new JWT(new Keys()).decode(data.issuedToken);
+    //const { sub } = new JWT(new Keys()).decode(data.issuedToken);
 
     const payload = JSON.stringify(claimData);
-    this.nats.connection.publish(
-      `${data.requester}.${NATS_EXCHANGE_TOPIC}`,
-      payload,
-    );
-    this.nats.connection.publish(`${sub}.${NATS_EXCHANGE_TOPIC}`, payload);
+    this.logger.debug(`publishing claims issue request ${payload} from ${did}`);
+    await this.claimProcessor.enqueueClaimIssuanceRequest(claimData);
```

Comment:
@Passerino What are the advantages/disadvantages of the queue approach in your view? I think the queue does make sense for the asynchronous portion (i.e. the messaging via NATS). However, I'm not entirely convinced that it makes sense to do the database updates via queues. I think if a database update failed for instance, I think it could be a bit odd that we returned a successful HTTP code. What are you thoughts on this?